fecgov/fec-cms
# Feature/3514 presidential map 1

## Summary
- Resolves #3513 
- Resolves #3514 
- Closes #3507  
There's still quite a bit to do—especially with styles (size, color, flow, visibility, etc) but progress is, well, progressing.
Think of this as a check-in PR to get a feel for how it's going and to start verifying that data is working correctly.

## Impacted areas of the application
Added more components and functionality.
Also calling the official endpoints (hence closing 3507)

## Screenshots
![image](https://user-images.githubusercontent.com/26720877/74378191-8095b100-4db3-11ea-9452-24c5550e48d1.png)

## Related PRs
#3514 

## How to test
- pull branch
- Set feature flag: `export FEC_FEATURE_PRESIDENTIAL_MAP=True`
- `npm i`
- `npm run build`
- `./manage.py runserver`
- check http://127.0.0.1:8000/data/candidates/president/presidential-map/
- Play with the year toggle, click candidates

The left list should change when the years are clicked.
The map colors, breadcrumb, candidate details, and eventual right column should all update as candidates are changed.
The various download links should change their href values as candidates are changed (though the state download button doesn't toggle off.
____
